Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -182,7 +182,7 @@ open class BaseMapView: UIView, CameraViewDelegate {
                                                object: nil)
 
         if let validStyleURI = styleURI {
-            mapboxMap.__map.setStyleURIForUri(validStyleURI.absoluteString)
+            mapboxMap.__map.setStyleURIForUri(validStyleURI.path)
```

Comment:
This will conflict with changes in https://github.com/mapbox/mapbox-maps-ios/pull/285; should be easy to resolve though